helmfile.yaml

```diff
@@ -27,7 +27,8 @@ bases:
   - helmfile/stacks/namespaces.yaml
 ---
 releases:
-  - inherit: [ template: ck8s-namespaces ]
+  - inherit: [ template: system-namespaces ]
+  - inherit: [ template: user-namespaces ]
```

Comment:
Are we happy with "system" and "user" namespaces or is there an other wording scheme we should use? @aarnq

Reply:
I'm thinking `admin` and `dev` as that would make it consistent with the glossary.